### PR TITLE
Add notifications for success/failed when creating/updating/deleting …

### LINKED ISSUE #11915: Changing retention policy on bucket only updates if the time changes
a few things:
1. i'm not sure why the default seems to be 1h. It should be all 0s
2. switching from forever to periodically should update the bucket

kapture 2019-02-14 at 14 12 33

### PR DESCRIPTION
**Pull request by Palakp41**
…scrapers

Closes #11915 

_Briefly describe your proposed changes:_

  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

File: ui/src/shared/copy/v2/notifications.ts [hunk 1]
```diff
@@ -124,3 +124,33 @@ export const bucketDeleted = (bucketName: string): Notification => ({
   ...defaultSuccessNotification,
   message: `Bucket ${bucketName} was successfully deleted`,
 })
+
+export const scraperDeleteSuccess = (scraperName: string): Notification => ({
+  ...defaultSuccessNotification,
+  message: `Scraper ${scraperName} was successfully deleted`,
```
Comment (chnn) [inline ui/src/shared/copy/v2/notifications.ts]:
```suggestion
  message: `Scraper “${scraperName}” was successfully deleted`,
```

File: ui/src/shared/copy/v2/notifications.ts [hunk 2]
```diff
@@ -124,3 +124,33 @@ export const bucketDeleted = (bucketName: string): Notification => ({
...
+})
+
+export const scraperDeleteFailed = (scraperName: string): Notification => ({
+  ...defaultErrorNotification,
+  message: `Failed to delete scraper: ${scraperName}`,
```
Comment (chnn) [inline ui/src/shared/copy/v2/notifications.ts]:
```suggestion
  message: `Failed to delete scraper “${scraperName}”`,
```

File: ui/src/shared/copy/v2/notifications.ts [hunk 3]
```diff
@@ -124,3 +124,33 @@ export const bucketDeleted = (bucketName: string): Notification => ({
...
+})
+
+export const scraperCreateFailed = (): Notification => ({
+  ...defaultErrorNotification,
+  message: 'Failed to create scraper',
+})
+
+export const scraperUpdateSuccess = (scraperName: string): Notification => ({
+  ...defaultSuccessNotification,
+  message: `Scraper ${scraperName} was updated successfully`,
```
Comment (chnn) [inline ui/src/shared/copy/v2/notifications.ts]:
```suggestion
  message: `Scraper “${scraperName}” was updated successfully`,
```

File: ui/src/shared/copy/v2/notifications.ts [hunk 4]
```diff
@@ -124,3 +124,33 @@ export const bucketDeleted = (bucketName: string): Notification => ({
...
+})
+
+export const scraperUpdateFailed = (scraperName: string): Notification => ({
+  ...defaultErrorNotification,
+  message: `Failed to update scraper: ${scraperName}`,
```
Comment (chnn) [inline ui/src/shared/copy/v2/notifications.ts]:
```suggestion
  message: `Failed to update scraper “${scraperName}”`,
```

File: ui/src/shared/copy/v2/notifications.ts (hunk 1 above)
Comment (Palakp41) [reply ui/src/shared/copy/v2/notifications.ts]: Got it, will update! Thanks :)

**Pull request merged**